### PR TITLE
ELY-2297 Wrong path for Setup the JBoss Maven Repository.

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/ELY-2297

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ $ git clone git@github.com:wildfly-security/wildfly-elytron.git
 Setup the JBoss Maven Repository
 --------------------------------
 
-To use dependencies from JBoss.org, you need to add the JBoss Maven Repositories to your Maven settings.xml. For details see http://community.jboss.org/wiki/MavenGettingStarted-Users
+To use dependencies from JBoss.org, you need to add the JBoss Maven Repositories to your Maven settings.xml. For details see [MavenGettingStarted-Users](https://developer.jboss.org/docs/DOC-15169)
```
Comment (skyllarr) [inline README.md]: @RanabirChakraborty  Thanks for the PR! You can use spaces in [MavenGettingStarted-Users] for readability.

Comment (RanabirChakraborty) [reply README.md]: Thanks! did the changes.

**Review by skyllarr — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (skyllarr) [reply README.md]: Thank you

**Review by fjuma — APPROVED** [review]
Thanks @RanabirChakraborty!

**Pull request merged**